esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -2740,3 +2741,68 @@ bool CWsWorkunitsEx::onWUUpdateQueryEntry(IEspContext& context, IEspWUUpdateQuer
     }
     return true;
 }
+
+bool CWsWorkunitsEx::onWUGetNumFileToCopy(IEspContext& context, IEspWUGetNumFileToCopyRequest& req, IEspWUGetNumFileToCopyResponse& resp)
+{
+    try
+    {
+        Owned<IPropertyTree> result;
+        StringBuffer clusterName = req.getClusterName();
```

Comment:
This is one of those cases where "ClusterName" is confusing.  You seem to be using it as "Target" and then looking up the "ProcessClusters" but then only hitting the first process.

It might be better to have the parameter be ProcessCluster and just look up the IP for that process.

Files being copied isn't really a Target thing... it's per ProcessCluster.